modules/services/clipman.nix

```
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.clipman;
+
+in {
+  meta.maintainers = [ maintainers.jwygoda ];
+
+  options.services.clipman = {
+    enable = mkEnableOption "clipman, a simple clipboard manager for wayland";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.clipman;
+      defaultText = "pkgs.clipman";
+      description = "clipman derivation to use.";
+    };
+  };
```

Should be possible to simplify to
```suggestion
    package = mkPackageOption pkgs "clipman" { };
```